.travis.yml

```diff
@@ -1,19 +1,27 @@
 ---
 language: python
-python: "2.7.15"
```

Comment:
I think travis isn't running because of this change, as Travis doesn't know which CI should be executed.